### DISCUSSION TITLE
I want to say something about user feedback

### DISCUSSION THREAD

**Discussion by cubxxw** [langfuse/langfuse#2668] [Ideas]
### Describe the feature or potential improvement

I am curious about the current user feedback and I have read your documentation

+ [https://langfuse.com/docs/scores/user-feedback](https://langfuse.com/docs/scores/user-feedback)

Depending on the type of the application, there are different types of feedback that can be collected that vary in quality, detail, and quantity.

1. Explicit Feedback: Directly prompt the user to give feedback, this can be a rating, a like, a dislike, a scale or a comment. While it is simple to implement, quality and quantity of the feedback is often low.
2. Implicit Feedback: Measure the user's behavior, e.g., time spent on a page, click-through rate, accepting/rejecting a model-generated output. This type of feedback is more difficult to implement but is often more frequent and reliable.

The first point is basic needs, which I think you can meet, and I want to know how you think about the second point.

I'm currently looking to build the best user research system in the world, which I think we can integrate.

The second point is somewhat consistent with my previous idea. As a first-class product manager, you can know whether your product is good or not. You can go to see how a user uses their product every day and what they do on which pages. That's my expectation.

**Comment by marcklingen** [comment]
Using implicit feedback is generally a very good idea, you can add this to Langfuse via the API/SDKs.

Example source of this feedback:
- any form of human-in-the-loop UI where a natural person interacts with outputs and approves/corrects them one-by-one
- retention or outcome metrics which help you understand whether an application gets better or worse over time to correlate with prompt/model/system changes. More abstract, thus less signal on a single trace level.

If you have more specific questions, happy to help!
